src/__tests__/App.test.tsx

```diff
@@ -3,18 +3,24 @@ import { act } from 'react-dom/test-utils';
 
 import App from '../components/App';
 import { render } from './utils/setupTests';
-import { screen } from './utils/test-utils';
+import { screen, FetchMockSandbox } from './utils/test-utils';
 
 describe('App', () => {
+  beforeEach(() => {
+    (global.fetch as FetchMockSandbox).get(
+      'glob:https://treeherder.mozilla.org/api/project/*/push/*',
```

Comment:
You'll notice different ways of defining the mock in these files. I changed while I was writing them, and didn't settle on one unique way of doing it. I think that's OK, as each file can have different needs, but tell me what you think.